### ISSUE TITLE
HasRows MockDbDataReader

### ISSUE THREAD

**Issue by htw8441** [Apps72/Dev.Data#37]
Hi,

I think there is an error in MockDbDataReader. The documentation of DbDataReader say: HasRows return true if the result has 1 or more rows. MockDbDataReader.HasRows() return only true if more as one rows found.

Can you check this please.

Regards
Andreas

**Comment by dvoituron** [comment]
Hello. Thanks for your feedback. I'll check that.

**Comment by dvoituron** [comment]
Indeed... That's will be corrected in the next release. Thanks again.
